app/Aggregates/Nft/Repositories/NftRepository.php

```diff
@@ -29,4 +29,9 @@ public function get(NftId $nftId): Nft
     {
         return $this->retrieve($nftId);
     }
+
+    public function save(Nft $nft): void
+    {
+        $this->persist($nft);
+    }
```

Comment:
ℹ️ Somehow forgot to persist the aggregates in most places, so had to add this method to all of them.

I find there is something quite unnatural in having to persist the aggregate on top of calling its methods generating events. I naturally assume the latter does the former.